staff/joel-suarez/strings/concat.js

```diff
@@ -0,0 +1,15 @@
+delete String.prototype.concat;
+
+function concat() {
+    var concatenatedString = '';
+    for (var i = 0; i < arguments.length; i++) {
+        var arg = arguments[i];
+        for (var j = 0; j < arg.length; j++) {
+            concatenatedString += arg[j];
+        }
+    }
+    return concatenatedString;
+}
+
```

Comment:
What happens when we try to concatenate two numbers? 🤯 
Check the concat() behaviour in that case, it should throw an error.